chain/x/zoracle/internal/types/querier.go

```diff
@@ -23,28 +23,31 @@ func (u64a U64Array) String() string {
 
 type RequestInfo struct {
 	CodeHash    []byte            `json:"codeHash"`
-	ParamsHex   []byte            `json:"paramsHex"`
 	Params      RawJson           `json:"params"`
+	ParamsRaw   []byte            `json:"paramsRaw"`
```

Comment:
@evilpeach Note this field change. Will need to update `scan` once this is live.